frontend/src/AlertsManagement/AlertsContent.test.js

```diff
@@ -17,14 +19,58 @@ const styles = {
   }
 };
 
-describe("handleTabs", () => {
-  const tabLabels = {
-    UNRESOLVED: 0,
-    RESOLVED: 1,
-    ALL: 2,
-  };
+describe("componentDidMount", () => {
+  // Mock data for helper function return value.
+  const expectedAlerts = new Map();
+  expectedAlerts.set(1234567890123456, {
+    timestampDate: "Sun Dec 08 2019",
+    anomalies: [{
+      dimensionName: "Ramen", metricName: "Interest Over Time",
```

Comment:
nit: maybe `Ramen` and `Interest Over Time` can be `const` as you seem to use it a lot in this file

Reply:
I would say that this isn't necessary and doesn't make as much sense to make them `const` because it's the mock data... but I can change it if you think that's better

Reply:
no worries, you can keep it as it is